src/main/scala/no/ndla/learningpathapi/service/UpdateServiceComponent.scala

```diff
@@ -188,6 +269,53 @@ trait UpdateServiceComponent {
       }
     }
 
+    def addLearningStepV2(learningPathId: Long, newLearningStep: NewLearningStepV2, owner: String): Option[LearningStepV2] = {
+      optimisticLockRetries(10) {
```

Comment:
Ser denne funksjonen er basert på `addLearningStep` som også bruker `optimisticLockRetries`, men akkurat denne gir ikke så mye for meg, men det er nok best å ta med @kstigen som har laget den